tests/time_tests/src/timetests_helper/statistics_writer.h

```diff
@@ -19,6 +23,9 @@
 class StatisticsWriter {
 private:
   std::ofstream statistics_file;
+  std::map<std::string, std::pair<int, float>> time_structure;
```

Comment:
let's add comment what data is expected to be contained - like `// timer_name, tab number, duration